.travis.yml

```
@@ -1,16 +1,25 @@
 language: python
+dist: focal
```

People on Fossa get notifications to upgrade to Jammy -- might be worth upgrading now to save us any future headaches
```suggestion
dist: jammy
```